resources/public/api/conf/1.0/common/pathParameters.yaml

```diff
@@ -0,0 +1,41 @@
+components:
+  parameters:
+    nino:
+      name: nino
+      in: path
+      description: |
+        National Insurance number, in the format `AA999999A`
+      schema:
+        type: string
+        example: TC663795B
+      required: true
+    taxYear:
+      name: taxYear
+      in: path
+      description: |
+        The tax year to which the data applies.The start year and end year must not span two tax years. No gaps are allowed, for example, 2019-21 is not valid. The minimum tax year is 2021-22.
+      schema:
+        type: string
+        pattern: ^\d{4}-\d{2}$
+        example: 2022-23
+      required: true
+    businessId:
+      name: businessId
+      in: path
+      description: |
+        An identifier for the business, unique to the customer.<p>Must conform to the regular expression `^X[A-Z0-9]{1}IS[0-9]{11}$`
```

Comment:
I don't think the description needs to include the section about the regular expression, as it is already included with the pattern below.